### DISCUSSION TITLE
评论twikoo 开启后 页面加载报错无法进入

### DISCUSSION THREAD

**Discussion by foryouos** [theme-shoka-x/hexo-theme-shokaX#118] [Q&A] [answered]
评论使用的twikoo
云函数使用的Zeabur 部署 
部署 https://foryouos.zeabur.app/ 已经显示成功啦
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/58133210/2c38983a-0b02-4057-a8ce-a7cdd5a69412)

```sh
# https://twikoo.js.org
twikoo:
  enable: true
  link: "https://cdn.staticfile.org/twikoo/1.6.16/twikoo.all.min.js"
  mode: vercel # vercel或tencent
  envId: "https://foryouos.zeabur.app/" # twikoo环境ID,vercel填地址
  region:
```
每次运行时，将enable换成true 网页就无法打开，也hexo clean 了，hexo s 就一直打不开，控制台有一条命令行出现的报错
不知道是不是mode的问题，这个或，是只能填其中一个嘛，那个使用vercel 网络的问题吧，不科学上网打不开，就放弃改其它的了，然后遇到了这个不理解的问题，求助，辛苦开发者啦！帮我看一下呗

![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/58133210/fc30216d-c5d3-47b9-b4e0-5282323b0dbf)

![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/58133210/ea4e5e10-3a10-44d0-a9d3-4260098e4f6e)

**Comment by zkz098** [comment, marked as answer]
这个是js压缩的问题，有一定概率会出现
解决方法是根目录配置文件中minify配置，有一行注释是`如果js压缩报错请删除此行`，把这行删掉应该就能解决
Mode的Tencent和vercel实际上指腾讯云函数和其他部署方法，写主题的时候twikoo只支持vercel和腾讯云部署所以就用了这个名字，和vercel网络没有关系

**Comment by foryouos** [comment]
解决了，由于网络的原因，改用了 Waline 评论 ，使用云函数使用的Zeabur 部署 ，很重要的一点，在直接复制的时候，网络链接可能输入为https://foryouos.zeabur.app/    最好不可以加  `/`,  会出现报错，或者文章下面无法显示评论
